### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by nottherealalanturing**
Add Responsive styles and media query

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -557,3 +555,136 @@ form > button {
...
+    width: 100%;
+    margin-top: 12px;
+  }
+
+  .contact-me {
+    padding: 10% 25%;
+    border-top-left-radius: 160px;
+    background-image: url("../assets/images/footer-bg.svg");
+  }
+}
```
Comment (Whoistolu) [inline styles/style.css]: - [x] [REQUIRED] Great desktop design, however, you seem to have mistakenly skipped the pattern of the template provided. The position of the text and images on the card was interchanged after every new card. Your design has all its card images on the left, meanwhile, the next card should have it's an image on the right.
Please see the images below for proper explanation:
#### Your implementation:
Screenshot from 2022-02-17 19-10-50
#### Design Template Provided:
Screenshot from 2022-02-17 19-10-58

File: styles/style.css [hunk 2]
```diff
@@ -557,3 +555,136 @@ form > button {
...
 }
+
+@media only screen and (min-width: 992px) {
+  .nav__logo {
+    margin-left: 142px;
+  }
+
+  .nav .navbar {
+    margin-right: 142px;
+  }
```
Comment (Whoistolu) [inline styles/style.css:571]: - [x] [REQUIRED] Kindly note that the hamburger icon is not supposed to be seen on the desktop view. Please remove it and add "Portfolio", "About", "Contact".
#### Your implementation

#### Provided Template Design

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @nottherealalanturing,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @nottherealalanturing, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I think it will be better if you can decrease the margin between `Projects cards`, that will make your code more similar to the Figma design ^^

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nottherealalanturing** [comment]
@Hamzaoutdoors I don't understand what you mean by `Project cars`

**Comment by Hamzaoutdoors** [comment]
@nottherealalanturing if you compare your projects cards between your project and the design you will see that you have some extra margin between `cards` so it will be better to decrease the margin to stick to the design

**Pull request merged**